mirdata/datasets/indexes/phenicx_anechoic_index.json

```diff
@@ -0,0 +1,721 @@
+{
+  "version": 1,
+  "multitracks": {
```

Comment:
This structure is a bit different than what I'd envisioned. See the structure I was thinking in the issue [here](https://github.com/mir-dataset-loaders/mirdata/issues/325#issuecomment-722717442)

in short - I think it makes sense to have at the top level of the index "tracks" and "multitracks", and within each multitrack, we keep the list of tracks (by index) but keep all the individual track metadata separate. This way, if e.g. you only want to use the track level annotations and dont care about the mixtures, you can (e.g. for solo pitch tracking), but you still keep track of what tracks go with which multitracks. What do you think?

Reply:
except a few instrument (doublebass) there are no track level annotations. we have three levels:
level 1: audios: violin1.wav, violin2.wav, violin3.wav 
level 2: annotations violin.txt -> these are the annotations for violin.wav = violin1+2+3
level 3: the mix: violin.wav+flute.wav, basically the sum of all tracks

similarly we can have level 2: strings.wav which is a mix of violin1+2+3+doublebass+viola1+2

btw we should talk to Helena Cuesta. they want to add the choir dataset, which may be a simpler case.

Reply:
would it be ok to have an index with this structure:
```
 'version': 1,
  'tracks': {
     "beethoven-basson1": {'annoation': ('path/to/file', checksum), 'audio': ...}
     "mahler-basson" : {...}
     ...
  },
  'multitracks': {
    "beethoven": {
         'tracks': ['beethoven-bassoon1', 'beethoven-bassoon2', 'beethoven-clarinet', ...],    
         'audio': ('beethoven/mix/path', checksum), 'notes-bassoon': ('melody/annotation/path', checksum), 'notes-clarinet': ('melody/annotation/path', checksum)
     }
    "mahler": ...
  },

```

Reply:
@nkundiushuti that works for me! I'll propose one alternative idea, but I think both would work:

Since level 1 is just audio, what if we treat level 2 as the "tracks", merging everything from level 1 into level 2, eg.

```
 'version': 1,
  'tracks': {
     "beethoven-bassoon": {
          'audio_bassoon1': (path, checksum), 
          'audio_bassoon2': (path, checksum),  
          'notes': (path, checksum)
     }
     "mahler-bassoon" : {...}
     ...
  },
  'multitracks': {
    "beethoven": {
         'tracks': ['beethoven-bassoon', 'beethoven-clarinet', ...],    
         'audio': ('beethoven/mix/path', checksum)
     }
    "mahler": ...
  },
```